src/interactive/repl.ts

```diff
@@ -755,6 +755,34 @@ async function executeCell(shouldMove: boolean = false) {
     await evaluate(ed, cellrange, code, module)
 }
 
+
+async function debugRunCell(shouldMove: boolean = false) {
+    telemetry.traceEvent('command-executeCell')
+
+    const ed = vscode.window.activeTextEditor
+    if (ed === undefined) {
+        return
+    }
+
+    const doc = ed.document
+    const selection = ed.selection
+    const cellrange = currentCellRange(ed)
+    if (cellrange === null) {
+        return
+    }
+    const code = doc.getText(cellrange)
+
+    await startREPL(true, false)
+
+    if (shouldMove && ed.selection === selection) {
+        const isJmd = doc.languageId === 'juliamarkdown'
+        const nextpos = new vscode.Position(nextCellBorder(doc, cellrange.end.line + 1, true, isJmd) + 1, 0)
+        validateMoveAndReveal(ed, nextpos, nextpos)
+    }
+    const debugCode = 'using Debugger\nbreak_on(:error)\nDebugger.@run begin\n'+code+'\nend'
```

Comment:
I'm confused. Why would we load Debugger instead of using the built-in Debugger here?

Reply:
Don' t know how to do that. calling `    debuggerRun({code: code,filename: ed.document.fileName})` gives me errors